### DISCUSSION TITLE
Vue avec serveurs physiques et logiques

### DISCUSSION THREAD

**Discussion by etienne-64** [dbarzin/mercator#98] [Ideas]
Sauf erreur de ma part, je n'ai trouvé aucune représentation permettant de voir  les serveurs logiques (VM) portés par un serveur physique. C'est une "vue" inter-vue intéressante comme "serveur physique - serveur logique - application".

**Comment by dbarzin** [comment]
Le rapport : "Configuration des serveurs logiques" donne la liste des applications et des serveurs physiques liés.

**Reply by etienne-64** [reply]
Merci. J'en découvre un peu plus chaque jour. L'essayer c'est l'adopter.
